### ISSUE TITLE
Pressing fast o and tab while in Normal mode resulting <tab>

### ISSUE THREAD

**Issue by Komar0ff** [VSCodeVim/Vim#6996]
**Describe the bug**
in Normal mode, press `o` and press `tab` rapidly, instead of add a new line and tab, it create a new line with resulting `<tab>`

**To Reproduce**

1.  Make sure you are in Normal mode
2.  Quickly press `o`(or `O`) and `tab`

**Expected behavior**
Add a new line and tab

- Extension (VsCodeVim) version: `v1.21.6`
- VSCode version: `1.59.0`
- OS: `MacOS Big Sur 11.4`

**Comment by rcywongaa** [comment]
I don't think this was ever fully fixed even though it has been closed as fixed a couple of times

https://github.com/VSCodeVim/Vim/issues/730
https://github.com/VSCodeVim/Vim/issues/831
https://github.com/VSCodeVim/Vim/issues/1195

**Comment by X9VoiD** [comment]
Still having this issue.
